cli/src/args.rs

```diff
@@ -104,20 +105,30 @@ impl RgbArgs {
         }
 
         let provider = FsBinStore::new(stock_path.clone())?;
-        let mut stock = Stock::load(provider, true).map_err(WalletError::WalletPersist).or_else(|err| {
-            if matches!(err, WalletError::Deserialize(DeserializeError::Decode(DecodeError::Io(ref err))) if err.kind() == ErrorKind::NotFound) {
-                if self.verbose > 1 {
-                    eprint!("stock file is absent, creating a new one ... ");
+        let mut stock = Stock::load(provider, true)
+            .map_err(WalletError::WalletPersist)
```

Comment:
I think we can remove this `map_err` here and move it down, saving some clarity in the checks within `or_else`
```suggestion
```